frontend/src/metabase/visualizations/visualizations/SmartScalar/SettingsComponents/PeriodsAgoOptionComponent.styled.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "@emotion/styled";
+import { NumberInput } from "metabase/ui";
+
+export const NumberInputStyled = styled(NumberInput)`
+  input {
+    text-align: center;
+    padding: 0.25rem;
+  }
+`;
```

Comment:
Can't we do that with Mantine props? 

```tsx
<NumberInput p="0.25rem" ta="center" /> 
```

Reply:
Unfortunately this does not work, since the input is actually wrapped by multiple divs.